actors/builtin/multisig/multisig_actor.go

```diff
@@ -34,13 +34,41 @@ type MultiSigTransaction struct {
 
 type MultiSigActor struct{}
 
+type musigMethods struct {
+	Constructor                 int64
+	Propose                     int64
+	Approve                     int64
+	Cancel                      int64
+	ClearCompleted              int64
+	AddSigner                   int64
+	RemoveSigner                int64
+	SwapSigner                  int64
+	ChangeNumApprovalsThreshold int64
+}
+
+var Methods = musigMethods{1, 2, 3, 4, 5, 6, 7, 8, 9}
```

Comment:
Please reconcile this with builtin/methods.go so the concept is not duplicated (for the multisig actor).

Since in general we need these to be outside the individual actor packages, to support inter-dependent storage power/miner/market actors, I suggest putting this in builtin/methods.go and replacing the existing values there. Leave the Exports() here though.